js_task1/firstTask.js

```diff
@@ -0,0 +1,26 @@
+const input = document.querySelector(".first-task input");
+const pushButton = document.querySelector(".first-task button");
+const outArea = document.querySelector(".first-task .out-area");
+const symbol = " $ "; // output symbol
+
+function outputLine(symbolCount) {
+  outArea.innerHTML += `${symbol.repeat(symbolCount)} <br>`;
```

Comment:
Please, try to avoid using .innerHTML in future. Also you can rewrite it with creating new DOM elements.

Reply:
Done